server/lib/notifications/agents/discord.ts

```diff
@@ -177,6 +181,7 @@ class DiscordAgent
         ...(payload.extra ?? []).map((extra) => ({
           name: extra.name,
           value: extra.value,
+          inline: true,
```

Comment:
Because this can be any arbitrary data, I felt inlining it wouldn't make sense if we decided to attach longer messages.